xtask/src/flags.rs

```diff
@@ -32,9 +32,12 @@ xflags::xflags! {
         cmd release {
             optional --dry-run
         }
-        cmd promote {
-            optional --dry-run
+
+        cmd pull {
+            optional --commit refspec: String
         }
+        cmd push {}
```

Comment:
Naming is up to you, but I would find it confusing that this pulls/pushes *with the rusc repo* without giving an indication of that.

Reply:
Renamed to `rustc-pull` and `rustc-push`.